integration_tests/mockApis/csipApi.ts

```diff
@@ -573,6 +573,50 @@ const stubCsipRecordSuccessAwaitingDecisionNoInterviews = () => {
   })
 }
 
+const stubCsipRecordSuccessAwaitingDecisionLongProtectiveFactors = () => {
+  return stubFor({
+    request: {
+      method: 'GET',
+      urlPattern: '/csip-api/csip-records/02e5854f-f7b1-4c56-bec8-69e390eb8550',
+    },
+    response: {
+      status: 200,
+      headers: {
+        'Content-Type': 'application/json;charset=UTF-8',
+      },
+      jsonBody: {
+        ...csip,
+        status: 'AWAITING_DECISION',
+        referral: {
+          ...csip.referral,
+          investigation: {
+            interviews: [
+              {
+                interviewee: 'Another Person',
+                interviewDate: '2024-12-29',
+                intervieweeRole: { code: 'CODE', description: 'Witness' },
+                interviewText: 'some text',
+              },
+              {
+                interviewee: 'Some Person',
+                interviewDate: '2024-12-25',
+                intervieweeRole: { code: 'CODE', description: 'Foobar' },
+                interviewText: 'other stuff',
+              },
+            ],
+            staffInvolved: 'staff stafferson',
+            evidenceSecured: 'SomeVidence',
+            occurrenceReason: 'bananas',
+            personsUsualBehaviour: 'a great person',
+            personsTrigger: 'spiders',
+            protectiveFactors: 'a'.repeat(3000),
+          },
+        },
+      },
+    },
+  })
+}
+
```

Comment:
This is why I've modified the inject route to be able to merge objects - so that we don't need to create a whole new object like this just to change one property. 

Do you want to wait for my PR to go in first or should we make a remedial PR to change this and any other examples of this after?

Reply:
Let's make a remedial PR - I know there's more than one instance of this.